README.rst

```diff
@@ -110,14 +110,21 @@ Always add tests to your contribution — if you want to test it locally before
 
 ::
 
-  $ python -m unittest discover tests
+  # Run all tests
+  $ python setup.py test
+
+  # Run all unit tests
+  $ pytest tests/unit
```

Comment:
`pytest` is only installed for tests, so this is gonna fail. Have you tried an envvar approach to separate unit test from integration tests? Something like: `python setup.py test` for unit tests and `RUN_INTEGRATION_TESTS=1 python setup.py test` for integration tests?